spec/views/collections/_collection.html.erb_spec.rb

```diff
@@ -18,6 +18,11 @@
     expect(rendered).to have_css('h2 a', text: collection.name)
   end
 
+  it 'renders a paragraph with the description of the collection' do
+    render
+    expect(rendered).to have_css('p', text: collection.description)
+  end
+
```

Comment:
It looks like `collection.description` is optional, so maybe add a test for that? I'm only seeing the name as required -- https://github.com/sul-dlss/mirador_sul/blob/master/app/models/collection.rb#L2